proposals/collection-expression-arguments.md

```diff
@@ -573,4 +573,8 @@ Span<int> s = [with()]; // ok?
 
 ## Open questions
 
-None currently.
+`with(...)` is a breaking change in the language with `[with(...)]`.  Before this feature, it means a collection expression with one element, which is the result of calling the `with`-invocation-expression.  After this feature, it is a collection, which has arguments passed to it. 
```

Comment:
Let's clarify that this is bringing back up the question from https://github.com/dotnet/csharplang/blob/main/meetings/2025/LDM-2025-03-17.md#conclusion, since we have now decided on the final syntax of the feature. Also would be good mention that since dictionary expression parsing is _not_ limited to C# 15, it has odd impacts when combined with `with()`.

Reply:
Thanks for finding the reference. Is there a reason to think the previous decision may need to be revised?